packages/ui/src/services/component-config/component-config.ts

```diff
@@ -18,7 +19,12 @@ export const useComponentConfigProps = <T extends DefineComponent>(component: T)
     const localConfigProps = localConfig.value
       .reduce((finalConfig, config) => config[component.name] ? { ...finalConfig, ...config[component.name] } : finalConfig, {})
 
-    const props = { ...globalConfigProps, ...localConfigProps }
+    const presetName = instance.props?.preset || localConfigProps.preset || globalConfigProps.preset
+    const getPresetProps = () => globalConfig.value.componentsPresets?.[component.name]?.[presetName]
```

Comment:
Shouldn't we be able to set preset for all components?

For example global preset 
```
'green': { color: 'green', class: 'with-🐸' }
```

And use it both on VaInput and VaButton for example?

Reply:
BTW, as we do renaming and breaking, it might be cool to rename `all` to `global` as the latter sounds better. :)

Reply:
@m0ksem @asvae 
`componentsAll` currently just applies `props` to all components, 
`preset` is also a `prop` and we can set it there - it will work fine.

Though creating `presets` in `componentsAll` tab will duplicate `componentsPresets` logic and seems a bit weird because its main purpose is to just apply a `prop` `value`, not to create a new entity.

Reply:
I can extend `Presets` logic to have `all` option next. Probably in a separate task after the core changes.